src/main/java/racingcar/view/input/Input.java

```diff
@@ -0,0 +1,65 @@
+package racingcar.view.input;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import racingcar.view.input.error.InputError;
+import racingcar.view.input.error.InputIllegalArgumentException;
+
+public final class Input {
+
+    public static final Pattern REMOVE_REGEX_PATTERN = Pattern.compile("\\s");
+    public static final String DIVIDING_STANDARD = ",";
+    public static final int MAXIMUM_NAME_LENGTH = 5;
+    public static final int MINIMUM_NAME_LENGTH = 1;
+
+    public static List<String> inputNames() {
+        String name = Console.readLine();
+
+        return devideToList(name);
+    }
+
+    public static int inputTrialAmount() {
+        String trialAmount = Console.readLine();
+        validateNotBlank(trialAmount);
+
+        try {
+            return Integer.parseInt(trialAmount);
+
+        } catch (InputIllegalArgumentException e) {
+            throw new InputIllegalArgumentException(InputError.MUST_BE_NUMBER);
+        }
+    }
+
+    private static List<String> devideToList(String value) {
+        String removedSpaceValue = removeSpace(value);
+        List<String> dividedName = Arrays.stream(removedSpaceValue.split(DIVIDING_STANDARD)).toList();
+
+        validateNameLength(dividedName);
+
+        return dividedName;
+    }
+
+    private static String removeSpace(String value) {
+
+        return REMOVE_REGEX_PATTERN.matcher(value).replaceAll("");
+    }
+
+    private static void validateNameLength(List<String> names) {
+        long countNames = names.stream()
+                .dropWhile(name -> name.length() >= MINIMUM_NAME_LENGTH && name.length() <= MAXIMUM_NAME_LENGTH)
+                .count();
```

Comment:
혹시 스트림과 count를 사용한 이유가 어떤 게 있는지 궁금합니다! dropWhile이라는 것도 저는 새로 알게된 기능이라서 궁금하기도 하고요ㅎㅎ
이렇게 forEach로 구현한 것과 어떤 차이점을 두고자 하셨는지 궁금합니다!

```suggestion
    private static void validateNameLength(List<String> names) {
        for (String name : names) {
            if (isNameValideLength(name)){
                throw new InputIllegalArgumentException(InputError.EXCEEDED_MAXIMUM_ERROR, MAXIMUM_NAME_LENGTH,
                        MINIMUM_NAME_LENGTH);
            }
        }
    }

    private static boolean isNameValideLength(String name) {
        return name.length() >= MINIMUM_NAME_LENGTH && name.length() <= MAXIMUM_NAME_LENGTH;
    }
```

Reply:
사실 뭔가 `stream`으로 2주차에 이것저것 많이 해보고 싶은 마음으로 
작성한 코드입니다! 🤣
당시 찾아봤었는데 성능상으로는 차이가 없었던걸로 기억해요! 

다만 다른 분들께서 말씀해주신 대로
이후 개수를 비교하는 것이 아닌
Length에 맞지 않는 이름이 나오는 경우 바로 예외를 던지는 것이 좋았겠다는 생각이 드네요! 
좋은 의견 감사합니다! 🥰